resources/views/service/centreusers/edit.blade.php

```diff
@@ -69,6 +69,10 @@ class="{{ $errors->has('worker_centre') ? 'error' : '' }}" required>
             </div>
             <button type="submit" id="updateWorker">Update worker</button>
         </form>
+        <form role="form" id="deleteForm" class="styled-form" method="GET"
+            action="{{ route('admin.centreusers.delete', ['id' => $worker->id]) }}">
+          <button type="submit" id="deleteWorker">Delete worker</button>
```

Comment:
form should have `{{ csrf_field() }}` in here. it tells laravel to add a hidden input with a "_token" field for csrf protections